docs/imagecustomizer/api/configuration/config.md

```diff
@@ -53,36 +53,43 @@ Added in v0.3.
 
 ## previewFeatures [string[]]
 
-Enables experimental and preview features that are not yet generally available.
+Enables experimental and preview features that are not yet generally available.  
 Features listed under previewFeatures must be explicitly included in the Image
 Customizer configuration to enable their usage.
 
 Supported options:
 
 - `uki`: Enables the Unified Kernel Image (UKI) feature.
 
-  When this option is specified, The `os.uki` configuration becomes available. A
+  When this option is specified, the `os.uki` configuration becomes available. A
   valid `os.bootloader.reset` value of `hard-reset` is required when `os.uki` is
   configured.
 
   Added in v0.8.
 
-Example:
+  Example:
 
-```yaml
-os:
-  bootloader:
-    resetType: hard-reset
-  uki:
-    kernels: auto
-previewFeatures:
-- uki
-```
+  ```yaml
+  os:
+    bootloader:
+      resetType: hard-reset
+    uki:
+      kernels: auto
+  previewFeatures:
+  - uki
+  ```
 
-Added in v0.8.
+- `output-artifacts`: Enables the configuration for the output directory
+  containing the generated artifacts.
+
+  When this option is specified, the `output.artifacts` configuration becomes available.
+
+  See [Output Artifacts](./outputArtifacts.md) for more details.
+
+  Added in v0.XX.
```

Comment:
The next release is `v0.14`.

Reply:
Ah, I thought we change it when release.. lol